_rules/audio-or-video-avoids-automatically-playing-audio-x0paj4.md

```diff
@@ -0,0 +1,319 @@
+
+### Bibliography
+
+- [Understanding Success Criterion 1.4.2: Audio Control](https://www.w3.org/WAI/WCAG21/Understanding/audio-control.html)
+- [Accessible Multimedia](https://developer.mozilla.org/en-US/docs/Learn/Accessibility/Multimedia)
+- [Failure of Success Criterion 1.4.2 for absence of a way to pause or stop an HTML5 media element that autoplays](https://www.w3.org/WAI/WCAG21/Techniques/failures/F93)
+- [G170: Providing a control near the beginning of the Web page that turns off sounds that play automatically](https://www.w3.org/WAI/WCAG21/Techniques/general/G170)
+
+## Test Cases
+
+### Passed
+
+#### Passed Example 1
+
+This `audio` element has (in its default controls) an [instrument][] to pause, stop, or turn the audio volume off.
+
+```html
+<audio src="/test-assets/moon-audio/moon-speech.mp3" autoplay controls></audio>
+```
+
+#### Passed Example 2
+
+This `video` element has (in its default controls) an [instrument][] to pause it, an [instrument][] to stop it, and an [instrument][] to turn its audio volume off.
+
+```html
+<video autoplay controls>
+	<source src="/test-assets/rabbit-video/video.mp4" type="video/mp4" />
+	<source src="/test-assets/rabbit-video/video.webm" type="video/webm" />
+</video>
+```
+
+#### Passed Example 3
+
+This `video` element autoplays and has an [instrument][] to pause, stop, or turn the audio volume off.
+
+```html
+<head>
+	<style>
+		button {
+			color: #000;
+		}
+		button:hover {
+			cursor: pointer;
+			cursor: pointer;
+			background-color: grey;
+			color: white;
+		}
+	</style>
+</head>
+<body>
+	<div id="video-container">
+		<!-- Video -->
+		<video id="video" autoplay>
+			<source src="/test-assets/rabbit-video/video.mp4" type="video/mp4" />
+			<source src="/test-assets/rabbit-video/video.webm" type="video/webm" />
+		</video>
+		<!-- Video Controls -->
+		<div id="video-controls">
+			<button type="button" id="play-pause" class="play">Play</button>
+			<button type="button" id="mute">Mute</button>
+		</div>
+	</div>
+	<script src="/test-assets/80f0bf/no-autoplay.js"></script>
+</body>
+```
+
+### Failed
+
+#### Failed Example 1
+
+This `audio` element autoplays, lasts for more than 3 seconds, and does not have an [instrument][] to pause, stop, or mute the audio.
+
+```html
+<audio src="/test-assets/moon-audio/moon-speech.mp3" autoplay></audio>
+```
+
+#### Failed Example 2
+
+This `video` element's audio autoplays for longer than 3 seconds, and does not have an [instrument][] to pause, stop, or mute the audio
+
+```html
+<video autoplay>
+	<source src="/test-assets/rabbit-video/video.mp4" type="video/mp4" />
+	<source src="/test-assets/rabbit-video/video.webm" type="video/webm" />
+</video>
+```
+
+#### Failed Example 3
+
+This `video` element has an [instrument][] to pause, stop, or turn the audio volume off but the [instrument][] is not visible.
```

Comment:
The expectation doesn't ask that the control be visible. Unlike the following examples, I can see an argument for why this would fail 1.4.2.

Reply:
You are correct. And in the definition of instrument we have a note stating that rules using the definition should not check for issues related to 2.1.1 and 4.1.2 so that we don't report one element in two different rules for the same reason.

And I'm curious... what is your argument for failing 1.4.2 in this instance?